src/ray/gcs/gcs_server/gcs_actor_scheduler.cc

```diff
@@ -238,6 +252,16 @@ void GcsActorScheduler::LeaseWorkerFromNode(std::shared_ptr<GcsActor> actor,
           }
 
           if (status.ok()) {
+            if (reply.worker_address().raylet_id().empty() &&
```

Comment:
I don't see any code that add this logic. Did it exist from before? How did you find this issue?

Reply:
We cancel the actor creation request, so the raylet id of the returned reply is empty. There was no cancel logic before.